### DISCUSSION TITLE
[QUESTION] Does it support Knowledge Distillation?

### DISCUSSION THREAD

**Discussion by mushan09** [NVIDIA/Megatron-LM#1024] [Q&A]
Does it support Knowledge Distillation?

**Comment by dongs0104** [comment]
check Nemo: https://github.com/NVIDIA/NeMo/pull/9849 
they are already developed but not released yet, 
> 4. Experiments and Results
> We use the NVIDIA Megatron-LM framework [45] to implement our pruning and distillation algorithms for compression and retraining. 
https://arxiv.org/pdf/2407.14679
